tests/frontend/unit/__snapshots__/SearchModal.spec.js.snap

```diff
@@ -1,100 +1,19 @@
-      </div>
-      <h3 class="u-mt-0_25">
-        search.in
-      </h3>
-      <div class="max-h-12 w-full align-top overflow-auto u-mb">
-        <div class="layout--flush">
-          <dp-checkbox-stub datacy="searchModal:statementText" datadpvalidateerrorfieldname="" id="statementText" label="[object Object]" name="search_fields[]" valuetosend="1" class="layout__item u-1-of-2"></dp-checkbox-stub>
-          <dp-checkbox-stub datacy="searchModal:authorName" datadpvalidateerrorfieldname="" id="authorName" label="[object Object]" name="search_fields[]" valuetosend="1" class="layout__item u-1-of-2"></dp-checkbox-stub>
-          <dp-checkbox-stub datacy="searchModal:statementId" datadpvalidateerrorfieldname="" id="statementId" label="[object Object]" name="search_fields[]" valuetosend="1" class="layout__item u-1-of-2"></dp-checkbox-stub>
-          <dp-checkbox-stub datacy="searchModal:internId" datadpvalidateerrorfieldname="" id="internId" label="[object Object]" name="search_fields[]" valuetosend="1" class="layout__item u-1-of-2"></dp-checkbox-stub>
-          <dp-checkbox-stub datacy="searchModal:consideration" datadpvalidateerrorfieldname="" id="consideration" label="[object Object]" name="search_fields[]" valuetosend="1" class="layout__item u-1-of-2"></dp-checkbox-stub>
-          <!---->
-          <!---->
-          <!---->
-          <!---->
-          <!---->
-          <!---->
-          <!---->
-          <!----> <input type="hidden" name="search_fields[]" value="tagNames" checked="checked" aria-hidden="true" class="hidden">
-          <!---->
-        </div>
-      </div>
-      <div class="text-right"><button type="button" data-cy="searchModal:submitSearchAdvanced" class="btn btn--primary u-mr">
-          apply
-        </button><button data-cy="searchModal:resetSearchAdvanced" class="btn btn--secondary">
-          reset
-        </button></div>
-    </dp-modal-stub>
-  </portal-stub>
+"<div class="flex">
+  <!-- Search Field --><label class="relative u-m-0"><button class="btn-icns fa fa-search c-at__controls-input-button" data-cy="searchAssessmentWordButton"></button>
+    <dp-input-stub id="searchterm" autocomplete="" datacounter="" datacy="searchAssessmentWordField" datadpvalidateerror="" datadpvalidateerrorfieldname="" datadpvalidateif="" datadpvalidateshouldequal="" hasicon="true" label="[object Object]" name="search_word2" pattern="" placeholder="searchterm" preventdefaultonenter="true" rounded="full" type="text" width="w-12" modelvalue="" aria-label="search.assessment.table" value=""></dp-input-stub>
+  </label><!-- Advanced Search button to open modal --><button type="button" data-cy="searchAdvanced" class="btn--blank o-link--default inline-block u-m-0 u-p-0 u-ml-0_5">search.advanced</button><!-- Modal content -->
+  <dp-modal-stub arialabel="" contentbodyclasses="" contentclasses="u-4-of-8-wide u-2-of-3-desk-down" contentheaderclasses="" modalid=""></dp-modal-stub>
```

Comment:
shouldn't the inner part of the modal be in the snapshot like before? Or what do we want to ensure should not change unintended?

Reply:
maybe for the time being, we can leave it like so... But we should think about what we want to archive with snapshot-tests.

Reply:
To render the default slot in vue 3 tests, we need to pass 
```
global: {
  renderStubDefaultSlot: true
}
```

Done in 917742bccf4a49c746433bc9fa7e26ac51f18cf7